src/cli/onefuzz/api.py

```diff
@@ -146,6 +147,7 @@ class Files(Endpoint):
 
     endpoint = "files"
 
+    @cached
```

Comment:
@bmc-msft is there any suggested ttl for it?

Reply:
Given the common usage of the Files endpoint, I think adding a cache of 1 hour is fine.

Since ttl for memoization.cached is specified in seconds, please add a constant ONE_HOUR_IN_SECONDS and use that.